wcs-importer-exporter.php

```diff
@@ -50,6 +50,9 @@ public static function init() {
 	 */
 	public static function setup_importer() {
 
+		if ( class_exists( 'WC_Subscriptions' ) && version_compare( WC_Subscriptions::$version, '2.0', '>=' ) ) {
```

Comment:
This check is unnecessary. The filter will only be called with Subscriptions 2.0, and when Subscriptions is active, so it's safe to just add the filter.